diff -Nur old/squidview.cpp new/squidview.cpp
--- old/squidview.cpp	2003-11-21 03:25:58.000000000 +0000
+++ new/squidview.cpp	2003-11-28 20:58:27.000000000 +0000
@@ -41,7 +41,7 @@
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #else
-#  define SHAREDIR "/usr/local/share/squidview"
+#  define SHAREDIR "/usr/share/doc/squidview-0.69/"
 #  define PACKAGE "squidview"
 #  define VERSION "0.6"
 #endif
@@ -3910,6 +3910,7 @@
   IOstring ("sRepFocus", sRepFocus);
   IOstring ("sRepBDTuser", sRepBDTuser);
   IOstring ("sViewer", sViewer);
+  IOstring ("sZipViewer", sZipViewer);
   IOstring ("sCurrentReport", sCurrentReport);
   IOstring ("sCurrentEmail", sCurrentEmail);
   IOint ("iSizeHitGrade", iSizeHitGrade);
@@ -3960,7 +3961,7 @@
     }
   }
   else
-    RunProgram (sViewer + " " + sHowToLink);
+    RunProgram (sZipViewer + " " + sHowToLink);
 }
 
 
diff -Nur old/squidview.h new/squidview.h
--- old/squidview.h	2003-11-21 03:25:58.000000000 +0000
+++ new/squidview.h	2003-11-28 20:55:09.000000000 +0000
@@ -76,12 +76,13 @@
 const char szWordFile[] = "words";         // search words 
 const char szUsersFile[] = "users";        // list of real names 
 const char szAliasesFile[] = "aliases";    // map IPs to user names
-const char szHowToFile[] = "HOWTO";      // link name 
+const char szHowToFile[] = "HOWTO.gz";      // link name 
 
 string sLogFile1, sLogFile2, sLogFile3;
 string sWordFile, sUsersFile, sAliasesFile, sSetFileName, sHowToLink;
 
 string sViewer = "less";
+string sZipViewer = "zless";
 string sCurrentReport = "";
 string sCurrentEmail = "";
 const string sReportExt = ".txt";
