--- squidview-0.30/squidview.h~	Sun Nov  3 19:53:13 2002
+++ squidview-0.30/squidview.h	Sun Nov  3 19:53:13 2002
@@ -56,7 +56,7 @@
 string sHomeDir;
 const char szSquidHome[] = ".squidview";
 string sPathToFiles;
-const char szMainLog[] = "/usr/local/squid/var/logs/access.log";
+const char szMainLog[] = "var/log/squid/access.log";
 const char szReadme[] = "/usr/share/doc/squidview/README";
 
 const char szLog1[] = "log1";
